src/app/components/tag/TagInput.js

```diff
@@ -106,19 +108,20 @@ class TagInput extends React.Component {
             onKeyPress={this.handleKeyPress.bind(this)}
             placeholder={this.props.intl.formatMessage(messages.search)}
           />
-          <Button
-            style={{ marginLeft: 'auto' }}
-            onClick={this.handleSubmit.bind(this)}
-            color="primary"
-            disabled={this.state.submitDisabled}
-          >
-            <FormattedMessage
-              id="tagInput.addTag"
-              defaultMessage="Add Tag"
-            />
-          </Button>
+      <Box ml="auto">
+            <Button
```

Comment:
I'm so sorry if I'm not being clear :sweat_smile: :sweat_smile: but `<Box` here is supposed to be aligned with the start of `<TextField` on line 102, since they're on the same level of nesting, right?

E.g.:

```
          <TextField
            id="tag-input__tag-input"
            value={this.state.value}
            error={this.state.message}
            helperText={this.state.message}
            onChange={this.handleChange.bind(this)}
            onKeyPress={this.handleKeyPress.bind(this)}
            placeholder={this.props.intl.formatMessage(messages.search)}
          />
          <Box ml="auto">
            <Button
```

Reply:
Aligned with textfield

Reply:
Wooo, no eslint errors anymore! :raised_hands::tada:

I'm now running unit and integration tests, which I expect to pass, then it will be ready to merge! :smile: